applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -611,6 +670,13 @@ The application supports the following configurations:
      - Release version of the application.
 
        Enables only the necessary application functionalities to optimize its performance.
+   * - Matter bridge over Wi-Fi
+     - :file:`prj_nrf70ek.conf`
```

Comment:
This probably could be removed.

Reply:
For an empty entry in a table, you can use `---` as the value.

Reply:
@kg-nordicsemi could you address this one?

Reply:
It turned out that the prj_nrf70ek.conf file is unnecessary when it comes to building, so I removed it, but I forgot to make changes to the documentation, I'll change it now